exposeForTests.go

```diff
@@ -0,0 +1,24 @@
+package janus
```

Comment:
If you rename this file to end in `_test.go`, it will still expose the methods to other packages but it won't be compiled into the binary 🙂  (see https://golang.org/pkg/cmd/go/#hdr-Compile_packages_and_dependencies)

Reply:
I need it to be in the janus package to have access to the private parameters on the Session and Handle, I'm using it from LS. I could nest it like our aws client tests tho if you'd like

Reply:
dw that doesn't work

Reply:
Nah, I meant literally rename the file (see the link).

Reply:
Sorry, I missed where you said it was for LS. Resolving as discussed.